### DISCUSSION TITLE
Single Collector Support How Many Agents

### DISCUSSION THREAD

**Discussion by AjinThankachan** [apache/skywalking#8663] [Q&A]
Hi,
I wish to know how many agents(Different or same language) can be supported by a single collector. And if we are implementing it for around 1000 servers(Agents) and One collector. What system requirements may require. What are the criteria for the specification? Just wish to know the details. If anyone can help it's very great. And advance thanks. 
Thanks

**Comment by wu-sheng** [comment]
Difference agent implementations, traffic, plugins, report channels, parameters mean this question is impossible to answer.
